build.sbt

```diff
@@ -94,8 +94,7 @@ lazy val commonSettings = ReleasePlugin.extraReleaseCommands ++ Seq(
   ),
   EclipseKeys.createSrc := EclipseCreateSrc.Default + EclipseCreateSrc.Resource,
   unmanagedClasspath in Test ++= Seq(
-    baseDirectory.value / "src" / "test" / "resources",
-    baseDirectory.value / "src" / "test" / "resources" / "sql"
```

Comment:
Out of curiosity, I had to add that line to make the H2 tests work. I wonder why they are now working :/